tools/dockerfile/test/bazel/Dockerfile

```diff
@@ -15,7 +15,7 @@
 FROM gcr.io/oss-fuzz-base/base-builder
 
 # Install basic packages and Bazel dependencies.
-RUN apt-get update && apt-get install -y software-properties-common python-software-properties
+RUN apt-get update && apt-get install -y software-properties-common python-software-properties python-dev
```

Comment:
Our Dockerfiles are generated from template files because we have so many of them and a lot of them share common package installations. Can you make this change to https://github.com/grpc/grpc/blob/master/templates/tools/dockerfile/test/bazel/Dockerfile.template and then run `./tools/buildgen/generate_projects.sh`? The script will update this Dockerfile for you.

Reply:
In fact, it's probably better if we use the standard Python installation template. You can replace that whole line in the template file with `<%include file="../../python_deps.include"/>`. Also, could you move this line to be right before `RUN apt-get install --reinstall python-pkg-resources`? Keeping the Python related installations together makes the Dockerfile more organized :)